data/ignition/files/usr/local/bin/start-cluster-installation.sh

```diff
@@ -5,7 +5,7 @@ source common.sh
 
 wait_for_assisted_service
 
-BASE_URL="http://{{.NodeZeroIP}}:8090/api/assisted-install/v2"
+BASE_URL="{{.ServiceBaseURL}}/api/assisted-install/v2"
```

Comment:
This results in an extra `/` as `BASE_URL="http://192.168.111.80:8090//api/assisted-install/v2`

Reply:
Fixed, thanks.